### ISSUE TITLE
swagger-client 3.8.26: Cannot find module 'traverse'

### ISSUE THREAD

**Issue by kevinoid** [swagger-api/swagger-js#1442]
### Q&A (please complete the following information)
 - OS: Linux
 - Environment: Node v10.15.2
 - Method of installation: npm
 - Swagger-Client version: 3.8.26
 - Swagger/OpenAPI version: Any

### Describe the bug you're encountering

Loading the `swagger-client` module throws an exception.

### To reproduce...

Steps to reproduce the behavior, run the following commands:

```sh
cd "$(mktemp -d)"
npm install swagger-client
node -e 'require("swagger-client")'
```

The last command will print the following output:

    internal/modules/cjs/loader.js:583
        throw err;
        ^
    
    Error: Cannot find module 'traverse'
        at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
        at Function.Module._load (internal/modules/cjs/loader.js:507:25)
        at Module.require (internal/modules/cjs/loader.js:637:17)
        at require (internal/modules/cjs/helpers.js:22:18)
        at Object.<anonymous> (/tmp/tmp.sgmmSX7qKt/node_modules/swagger-client/dist/index.js:1:32164)
        at t (/tmp/tmp.sgmmSX7qKt/node_modules/swagger-client/dist/index.js:1:177)
        at Object.<anonymous> (/tmp/tmp.sgmmSX7qKt/node_modules/swagger-client/dist/index.js:1:16949)
        at t (/tmp/tmp.sgmmSX7qKt/node_modules/swagger-client/dist/index.js:1:177)
        at Object.<anonymous> (/tmp/tmp.sgmmSX7qKt/node_modules/swagger-client/dist/index.js:1:30548)
        at t (/tmp/tmp.sgmmSX7qKt/node_modules/swagger-client/dist/index.js:1:177)

### Expected behavior

The `swagger-client` module can be loaded without throwing an exception.

**Comment by shockey** [comment]
Thanks for the heads-up!

I noticed this shortly after you did, while running the release job for Swagger UI: https://jenkins.swagger.io/job/oss-swagger-ui-release/47/

I’ve unpublished 3.8.26 and released 3.9.0, which correctly declares `traverse` as a production dependency.

**Comment by kevinoid** [comment]
Thanks @shockey!
